device.go

```diff
@@ -340,7 +340,10 @@ type BlockDevice struct {
 	DeviceType string
 	DeviceInfo DeviceInfo
 
-	// Path at which the device appears inside the VM, outside of the container mount namespace.
+	// SCSI Address of the block device, in case the device is attached using SCSI driver
```

Comment:
Could you add details of the format here as you did in https://github.com/clearcontainers/agent/pull/205?

Reply:
I have added details here.